frontend/package.json

```diff
@@ -42,6 +42,7 @@
     "precypress:run:junit": "npm run cypress:delete:reports",
     "cypress": "cypress open",
     "cypress:run": "cypress run",
+    "cypress:ci": "cypress run -e TAGS=\"not @crd-validation\" && cypress run -e TAGS=\"@crd-validation\"",
     "cypress:run:junit": "cypress run --reporter cypress-multi-reporters --reporter-options configFile=reporter-config.json",
```

Comment:
I'm not sure to understand :-)
Do you want the `cypress:run*` commands to run these new tests at last? Or do you want them to not run these new tests?

If you want the former, I think the `cypress:run` would become the same as `cypress:ci`.

Reply:
> If you want the former, I think the cypress:run would become the same as cypress:ci

Yeah now that you mention it, you can just update `cypress:run` and `cypress:run:junit` without needing to add a `cypress:ci` command. `cypress:run` is intended to be used locally to make sure all tests pass before pushing to upstream so it should run the same tests as what is in CI. I believe the OCP pipeline runs `cypress:run:junit` so that should also run the same tests.

Reply:
Done.